BlinkID/lib/microblink_scanner_widget.dart

```diff
@@ -19,6 +19,7 @@ class MicroblinkScannerWidget extends StatefulWidget {
     required this.onError,
     required this.onFirstSideScanned,
     required this.onDetectionStatusUpdate,
+    required this.mirrorAndroidFrontCameraPreview,
```

Comment:
Why do you need to pass additional parameters?

Reply:
I think we still need to choose how we will solve this particular issue. But my idea was to have some logic like FeatureFlag and then switch mirroring on/off for specific devices so this parameter would be required.